recipes-core/lightdm-fb/files/postinst

```diff
@@ -0,0 +1,12 @@
+#
+# Copyright (c) Siemens AG, 2023
+#
+# This file is subject to the terms and conditions of the MIT License.  See
+# COPYING.MIT file in the top-level directory.
+#
+
+# Add support for hotplug of Display Port
+sed -i 's/LABEL="systemd_end"/SUBSYSTEM=="graphics", KERNEL=="fb0", SYMLINK="fb0", TAG+="systemd"\nLABEL="systemd_end"/g' /lib/udev/rules.d/99-systemd.rules
```

Comment:
Please ship and install a file to /etc. Make this package a customization package.

Reply:
in addition, the whole purpose of rules.d is to let people add their own rules (in separate files)

Reply:
The correct debian way would be to use `dh_installudev`. By that, the file also gets correctly numbered. Just put it into its own package and add a file `<package>.udev` in the debian folder. For details, see `man dh_installudev`.